cmd/ctr/commands/containers/restore.go

```diff
@@ -87,11 +90,53 @@ var restoreCommand = cli.Command{
 			topts = append(topts, containerd.WithTaskCheckpoint(checkpoint))
 		}
 
-		task, err := ctr.NewTask(ctx, cio.NewCreator(cio.WithStdio), topts...)
+		useTTY, err := containerd.CheckSpecTTY(ctx, client, checkpoint.Target())
```

Comment:
Never used the criu support here, but does the container object returned from `client.Restore()` have its spec filled in as normal? If so could we just check the spec returned from `ctr.Spec()` and see if terminal is set?

Reply:
Ugh github didn't select all of the lines I did -_-. I had lines 83-93 selected if that makes the comment easier to parse.

Reply:
You are correct. I have removed the unnecessary call to `checkSpecTTY()`.